addons/launcher-template/launcher-template.addon

```diff
@@ -0,0 +1,4 @@
+# Name: launcher-template
+# Description: Imports the launch.openshift.io template
+
```

Comment:
And try to version using the same versioning schema as origin. My recommendation is to use branches and not tags, as branches can be updated. Then you can tag a specific item if required. [Here's](https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files/examples) an example repository doing that